IntegrationTests/DataImport/CSVImporterIntegrationTests.swift

```diff
@@ -23,14 +23,14 @@ import BrowserServicesKit
 
 final class CSVImporterIntegrationTests: XCTestCase {
 
-    override func setUpWithError() throws {
+    override func setUp() {
         super.setUp()
-        try clearDB()
+        try? clearDB()
```

Comment:
I don't necessarily want these to fail the test. So just letting them fail silently and hoping that other tests don't fail as a result.